examples/verlet_chain/chain.py

```diff
@@ -1,16 +1,17 @@
 import pyqtgraph as pg
+from pyqtgraph.Qt import QtCore
 import numpy as np
 import time
 from . import relax
 
 
-class ChainSim(pg.QtCore.QObject):
+class ChainSim(QtCore.QObject):
     
-    stepped = pg.QtCore.Signal()
-    relaxed = pg.QtCore.Signal()
+    stepped = QtCore.Signal()
+    relaxed = QtCore.Signal()
```

Comment:
I have no actual objection to this change, but having this as 
`pg.QtCore` might be a useful reminder that this version of QtCore has been touched by the import into PyQtGraph.
In this specific case, the code relies on the patched-in alias ` QtCore.Signal = QtCore.pyqtSignal` when running on PyQt.

Where that isn't the case and only the normal functionality is used, simplifying pg.QtCore to QtCore seems like a good idea.

Reply:
I have created fake QtCore, QtGui, QtWidgets so that pyqtgraph's monkeying of the module attributes are local.
In particular, ```QtCore.Signal = QtCore.pyqtSignal``` is now a local change.

Monkey-patching in methods to classes will still modify the actual classes, but most of those are deprecated and slated for removal in 0.13.0